src/program.js

```diff
@@ -601,6 +601,11 @@ Example: $0 --help run.
             'Only used with `use-submission-api`',
           type: 'string',
         },
+        'version-source': {
```

Comment:
Is --version-source the best public-facing option that we can think of?

Here are some other random options that I can think of that are more descriptive:

--submission-source=
--source-zip=
--private-source=

Reply:
well, it's the most accurate... it's the source for the version that's being created (that uploads to a field called `source` on the `/version/` endpoint) but if you think there's naming that would be more intuitive for web-ext users, sure.  Which one do you want?

Reply:
throwing in some more options:
--readable-source
--source-archive
--review-sources

Reply:
--upload-source-code= ? That is the label of the upload form at https://extensionworkshop.com/documentation/publish/source-code-submission/ and it's obvious that the source code is going to be uploaded somewhere.